app/views/content/train-to-be-a-teacher/what-is-qts.md

```diff
@@ -100,6 +100,6 @@ You can usually do this if you’ve worked as an unqualified teacher:
 
 ## Get QTS if you qualified as a teacher in Wales, Scotland or Northern Ireland
 
-If you trained to teach in Wales, your QTS will be automatically recognised in England and awarded by the [Education Workforce Council (EWC)](https://www.ewc.wales/site/index.php/en/).
+If you trained to teach in Wales, your QTS will be automatically recognised in England and awarded by the Education Workforce Council (EWC). Visit the [Education Workforce Council website to find out more](https://www.ewc.wales/site/index.php/en/).
```

Comment:
I don't think this link text makes sense in isolation for screen reader users `Education Workforce Council website to find out more` - think it needs further tweaking?